src/api/sizeme-api.js

```diff
@@ -492,6 +497,9 @@ function selectSize (size, auto) {
         } else {
             state = "no-fit";
         }
+        if (firstMatch && matchResult.recommendedFit === currentSize) {
+            dispatch(actions.selectSize({auto: true}));
+        }
```

Comment:
if this is the first match since page load and the pre-selected size was actually the recommended fit, change `auto` to true to enable the recommendation splash